docs/reference/online-stores/remote.md

```diff
@@ -0,0 +1,21 @@
+# Remote online store
+
+## Description
+
+This remote online store will let you interact with remote feature server. At this moment this only supports the read operation. You can use this online store and able retrieve online features `store.get_online_features`  from remote feature server. 
+
+## Examples
+
+The registry is pointing to registry of remote feature store. If it is not accessible then should be configured to use remote registry.
+
+{% code title="feature_store.yaml" %}
+```yaml
+project: my-local-project
+  registry: /remote/data/registry.db
+  provider: local
+  online_store:
+    path: http://localhost:6566
```

Comment:
shouldn't we have host and port fields instead? this is how we configured the remote offline:
```
offline_store:
    type: remote
    host: localhost
    port: 8815
```

Reply:
I have noticed this for offline_Store. We are already having path field for online store so reusing the same instead of introducing new properties. Remote registry is also doing the same. Also original recommendation by @tokoko but happy to change the code to keep it consistent across. 

https://github.com/feast-dev/feast/blob/master/sdk/python/feast/infra/registry/remote.py#L30

Reply:
I also simply avoided introducing new properties. Sounds more convenient as a single url to me, you always have some space to play with protocols and so on...